pkg/types/openstack/platform.go

```diff
@@ -62,6 +62,11 @@ type Platform struct {
 	// +optional
 	ClusterOSImage string `json:"clusterOSImage,omitempty"`
 
+	// ClusterOSImageProperties is a list of properties to be added to the metadata of the uploaded Glance ClusterOSImage.
+	// in the install-config.yaml. hw_qemu_agent=yes would look like
```

Comment:
Depending on whether it's possible or not to have different defaults for OKD / OCP, where we can assume the hypervisor is libvirt/kvm. 
```suggestion
	// Default: the default is to not set any properties.
```